.eslintrc

```diff
@@ -1,17 +1,15 @@
 {
   "root": true,
+  "env": {
+    "node": true,
+    "es2021": true
```

Comment:
why not 2022?

Reply:
<img width="790" alt="image" src="https://user-images.githubusercontent.com/104714497/179516021-ceff2670-3188-4bd7-ad1f-78f3fb02b8c3.png">

Reply:
according to eslint repo - we cant change it right now - it does not work or will effect - me and yarin have decided to leave it like that so the ecmaVersion should be coordinated with the node/typescript version